src/models/Position.ts

```diff
@@ -0,0 +1,4 @@
+export interface Position {
```

Comment:
JSDoc comments

Reply:
I will put them but let's discuss about this, I am not sure I have the same opinion about JSDoc than you 🙃

Reply:
Yes, lets discuss tomorrow